exercises/connect/connect_test.exs

```diff
@@ -0,0 +1,106 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("connect.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule ConnectTest do
+  use ExUnit.Case
+
+  def remove_spaces(rows) do
+    Enum.map(rows, &String.replace(&1, " ", ""))
+  end
+
+  @tag :pending
```

Comment:
Oh, we should also have one test where `@tag :pending` is commented out so the user has a place to start.